bindings/cpp/Makefile

```diff
@@ -30,7 +30,7 @@ ifeq "$(THREADING)" "pthread"
 override CPPFLAGS += -DGTEST_HAS_PTHREAD=1 -DWTF_PTHREAD_THREADED
 override LDLIBS += -pthread
 else ifeq "$(THREADING)" "std"
-override CPPFLAGS += -DGTEST_HAS_PTHREAD=1
+override CPPFLAGS += -DGTEST_HAS_PTHREAD=1 -DWTF_STD_THREADED
```

Comment:
hi Stella, thanks for the response. I was not sure about historical reasons thus tried to avoid modifying the default dehavior. Modifying the Makefile makes a lot more sense to me.